test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/NodeEndToEndTests.cs

```diff
@@ -231,7 +231,8 @@ public async Task Scenario_Logging()
             // verbose logs are filtered by default (the TestLogger explicitly
             // allows all levels for testing purposes)
             var key = MetricsEventManager.GetAggregateKey(MetricEventNames.FunctionUserLog, "Scenarios");
-            // Assert.Equal(9, Fixture.MetricsLogger.LoggedEvents.Where(p => p == key).Count());
+            //Assert.Equal(9, Fixture.MetricsLogger.LoggedEvents.Where(p => p == key).Count());
+            //Assert.Contains($"{MetricEventNames.HostStartupRuntimeLanguage}.node", Fixture.MetricsLogger.LoggedEvents);
```

Comment:
@brettsam -  Any ideas how to ovverride metrics logger for these tests? Test has a TODO
```// TODO: Re-enable once we can override IMetricsLogger```

Reply:
I have a PR out that enables this -- need to get some eyes on it though: https://github.com/Azure/azure-functions-host/pull/4109